packages/slice-machine/components/ChangesSectionHeader/ChangesSectionHeader.tsx

```diff
@@ -18,7 +18,6 @@ export const ChangesSectionHeader: React.FC<ChangesSectionHeaderProps> = ({
         backgroundColor: "#EEEEEE",
         borderRadius: 4,
         padding: "12px 16px",
-        mb: "16px",
```

Comment:
margin-bottom, I think the design people will like a look at this 😅

Reply:
I moved the margin-bottom from here to the `.changes.tsx` page since one component needed it but the other didn't